.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,44 @@
+<!--  Thanks for sending a pull request!  Here are some tips for you:
+
+1. If this is your first time, please read our contributor guidelines here:
+https://github.com/replicatedhq/kots/blob/master/CONTRIBUTING.md.
+2. Ensure you have added appropriate tests for your PR. For more information read here:
+https://github.com/replicatedhq/kots/blob/master/CONTRIBUTING.md#testing
+3. If the PR is unfinished, please mark it as a draft.
+-->
+
+#### What type of PR is this?
+
+<!--
+Please choose from one of the following:
+kind/bug
+kind/documentation
+kind/enhancement
+kind/chore
+-->
+
+#### What this PR does / why we need it:
+
+#### Which issue(s) this PR fixes:
+<!--
+*Automatically closes linked issue when PR is merged.
+Usage: `Fixes #<issue number>`, or `Fixes (paste link of issue)`.
+-->
+Fixes #
+
+#### Special notes for your reviewer:
+
+#### Does this PR introduce a user-facing change?
+<!--
+If no, just write "NONE" in the release-note block below.
+If yes, a release note is required:
+-->
+```release-note
+
+```
+
+#### Does this PR require documentation?
+<!--
+If no, just write "NONE" below.
+If yes, link to the related https://github.com/replicatedhq/kots.io documentation PR:
```

Comment:
I think it's reasonable to expect a kots.io pr from employees, but maybe we could request a link to a public gist instead?

Reply:
Let's just see how it goes...